.zshrc

```diff
@@ -18,7 +18,15 @@ plugins=(
 # Make all GIT repos into named directories.
 for repo in ~/git/*; do hash -d "$(basename "$repo")=$repo"; done
 
+# Make all workspaces into named directories.
+for workspace in ~/git/development/workspace/*; do
+    hash -d "$(basename "$workspace")=$workspace";
+done
+
 source $ZSH/oh-my-zsh.sh
 
 # Profile generated for docker-compose-development.
 [ -f "$HOME/.zshrc-development" ] && source "$HOME/.zshrc-development"
+
+# Run AWS CLI through Docker.
+alias aws='docker run --rm -it -v ~/.aws:/root/.aws -v $(pwd):/aws amazon/aws-cli'
```

Comment:
@JeroenBoersma is there anything you personally added on top of this?

Reply:
Yes I've... 

Here you go!

Reply:
https://gist.github.com/JeroenBoersma/87e29fd4aa06ec42216c80a6e3649fa5